meson.build

```diff
@@ -30,6 +30,7 @@ openrtx_src = ['openrtx/src/state.c',
                'openrtx/src/input.c',
                'openrtx/src/calibUtils.c',
                'openrtx/src/queue.c',
+               'openrtx/src/chan.c',
```

Comment:
This is an unbuffered channel. It provides a synchronous way to handle data between threads.

I've put it in here as it's a generic structure, but we can move it under the emulator folder if we want to conditionally include only for the linux platform.